src/home.js

```diff
@@ -1,168 +1,96 @@
-        <h5 class="card-title">Card title</h5>
-        <p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This content is a little bit longer.</p>
-        <p class="card-text">Last updated 3 mins ago</p>
-      </div>
-    </div>
-  </div>
-  <div class="col-lg-4 col-md-6 col-sm-12 col-xs-12 col-xl-4">
-    <div class="card bg-dark text-white">
-      <img class="card-img" src="https://i2.wp.com/4.bp.blogspot.com/-TN_IidPpIOg/UZOMDXggagI/AAAAAAAADC4/9N1zYIzlRT8/s400/Boiled+irish+potato+with+vegetable+sauce.jpg?resize=400%2C300" alt="Card image">
-      <div class="card-img-overlay">
-        <h5 class="card-title">Card title</h5>
-        <p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This content is a little bit longer.</p>
-        <p class="card-text">Last updated 3 mins ago</p>
-      </div>
-    </div>
-  </div>
-  <div class="col-lg-4 col-md-6 col-sm-12 col-xs-12 col-xl-4">
-    <div class="card bg-dark text-white">
-      <img class="card-img" src="https://allnigerianfoods.com/wp-content/uploads/jollof-rice.jpg" alt="Card image">
-      <div class="card-img-overlay">
-        <h5 class="card-title">Card title</h5>
-        <p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This content is a little bit longer.</p>
-        <p class="card-text">Last updated 3 mins ago</p>
-      </div>
-    </div>
-  </div>
-</div>
-
-</div>
-
-
-`;
-
-const renderHome = () => {
-  content.insertAdjacentHTML('beforeend', slider);
-  content.insertAdjacentHTML('beforeend', aboutInfo);
-};
-
-export default renderHome;
\ No newline at end of file
+const landingPageImg = 'https://images.unsplash.com/photo-1504674900247-0877df9cc836?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=750&q=80';
+
+// Initial load for Home tab
+const initialLoad = () => {
+  let homeMainContainer = document.createElement('div');
+  homeMainContainer.setAttribute('id', 'home-main-container');
+
+  // elements that belong to mainContainer
+  let landingImg = document.createElement('img');
+  landingImg.setAttribute('src', landingPageImg);
+  landingImg.setAttribute('id', 'landing-img');
+  landingImg.setAttribute('alt', 'landing-page');
+
+  let phraseBox = document.createElement('div');
+  phraseBox.setAttribute('id', 'phrase-box');
+
+  let mainPhrase = document.createElement('p');
+  mainPhrase.setAttribute('id', 'main-phrase');
+  mainPhrase.innerHTML = 'WELCOME TO TASTY BITE';
+
+  // Appending elements to the Main
+  phraseBox.appendChild(mainPhrase);
+  homeMainContainer.appendChild(landingImg);
+  homeMainContainer.appendChild(phraseBox);
+
+  return homeMainContainer;
+}
+
+// Function for the initial setup that's repeated again and again
+const repeatedSetup = () => {
+  /**
+   * 1. Making all elements first that are in 'index.html'
+   */
```

Comment:
Too much of comment make code look unprofessional, your code is well written and understandable.
kindly remove this comment.